Survey/Survey/Sources/Supports/Helper/Typealiases/Typealiases.swift

```diff
@@ -9,3 +9,6 @@
 import Foundation
 
 typealias ImageAssets = R.image
+typealias ColorAssets = R.color
+typealias FontAssets = R.font
+typealias LocalizedTextAssets = R.string
```

Comment:
This is code from Worpt, and I think it can save you a lot of time for typing localizable strings
```suggestion
typealias Localize = R.string.localizable
```

Reply:
update in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41/commits/f10d87bdfe1267e19fe24bb94768bd66860444a7)